src/components/RatingsReviews/RatingsReviews.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import ReviewsList from './ReviewsList.jsx';
+import AddReview from './AddReview.jsx';
+import RatingBreakdown from './RatingBreakdown.jsx';
+import ProductBreakdown from './ProductBreakdown.jsx';
+import { API_KEY } from  '../../../src/config/config.js';
+
+const axios = require('axios');
+
+class RatingsReviews extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      reviews: [],
+      sort: 'relevant',
+      addReviewModal: false
+    }
+    this.handleSort = this.handleSort.bind(this);
+    this.toggleReviewModal = this.toggleReviewModal.bind(this);
+  }
+
+  componentDidMount() {
+    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/reviews', {
+      headers: {'Authorization': `${API_KEY}`},
+      params: {
+        count: 2,
+        page: 1,
+        product_id: 66673, // FIXME - update to passed-in product_id
```

Comment:
just a reminder to do this li'l FIXME :)

Reply:
Fixed all the placeholders for the product ID! Had to refactor to a ComponentDidUpdate and everything is still working on my end!